lab-dustin/lib/greet.js

```diff
@@ -0,0 +1,8 @@
+function greetFunction(name) {
```

Comment:
We very, very rarely put things' type in their name - `greet`, not `greetFunction`!